packages/odyssey-react/src/components/Text/Text.module.scss

```diff
@@ -349,3 +357,54 @@
 .fontLineHeight {
   line-height: 1;
 }
+
+// Hover
+:hover > .bodyHoverColor,
+.bodyHoverColor:hover {
+  color: $text-body;
+}
+:hover > .bodyInverseHoverColor,
+.bodyInverseHoverColor:hover {
+  color: $text-body-inverse;
+}
+:hover > .primaryDarkHoverColor,
+.primaryDarkHoverColor:hover {
+  color: $color-primary-dark;
+}
+
+// Focus
+:focus > .bodyFocusColor,
+.bodyFocusColor:focus {
+  color: $text-body;
+}
+
+:focus > .bodyInverseFocusColor,
+.bodyInverseFocusColor:focus {
+  color: $text-body-inverse;
+}
+
+:focus > .primaryFocusColor,
+.primaryFocusColor:focus {
+  color: $color-primary-base;
+}
+
+// Disabled
+:disabled > .bodyInverseDisabledColor,
+.bodyInverseDisabledColor:disabled {
+  color: $text-body-inverse;
+}
+
+:disabled > .primaryLightDisabledColor,
+.primaryLightDisabledColor:disabled {
+  color: $color-primary-light;
+}
+
+:disabled > .subDisabledColor,
+.subDisabledColor:disabled {
+  color: $text-sub;
+}
+
+// Transition
+.inheritTransition {
+  transition: inherit;
+}
```

Comment:
this seems reasonable to me - while there are ways to recreate some of this as a JS API I don't believe we should so I"m in favor of these type of pseudo-selector styles.

One thing we'll want to add here are story examples so we have VRT testing for this - it may require a bit of javascript on your part to force the pseudo-selector state in the DOM [perhaps similar to this example I wrote.](https://github.com/okta/odyssey/blob/67a8120e4ab36107c7182fa1c501ea6f43ed02fd/packages/odyssey-storybook/src/components/Checkbox/Checkbox.stories.tsx#L65-L73)